include/domain/impl/receivers/kernel.tpp

```diff
@@ -84,6 +84,10 @@ void specfem::domain::impl::kernels::receiver_kernel<
       NGLL, DimensionType, MediumTag, specfem::kokkos::DevScratchSpace,
       Kokkos::MemoryTraits<Kokkos::Unmanaged>, true, true, true, false, using_simd>;
 
+  using Aux2ComponentFieldType = specfem::element::field<
```

Comment:
I think what you might want to use is `using Aux2ComponentFieldType = typename ElementFieldType::ViewType`. This should directly give you access to the underlying Kokkos::View used to store the data

Reply:
Can we remove `Aux2ComponentFieldType` if its not getting used?